corehq/apps/commtrack/models.py

```diff
@@ -193,6 +157,7 @@ class CommtrackConfig(QuickCachedDocumentMixin, Document):
     multiaction_enabled = BooleanProperty()
     multiaction_keyword_ = StringProperty()
 
+    # todo: remove?
     requisition_config = SchemaProperty(CommtrackRequisitionConfig)
     openlmis_config = SchemaProperty(OpenLMISConfig)
```

Comment:
The one I'd come across was v0.3 of ProductAPI:
https://github.com/dimagi/commcare-hq/pull/12787#discussion_r74426280